lib/onc_certification_g10_test_kit/profile_selector.rb

```diff
@@ -57,6 +74,31 @@ def guess_profile(resource) # rubocop:disable Metrics/CyclomaticComplexity
           return extract_profile('HeadCircumference') if observation_contains_code(resource, '9843-4') # rubocop:disable Style/SoleNestedConditional
         end
 
+        if Feature.us_core_v4? # New profiles in us core v5
+
+          return extract_profile('ObservationClinicalTest') if suite_options[:us_core_version] == 'us_core_5' &&
+                                                               resource_contains_category(
+                                                                 resource, 'clinical-test', 'http://terminology.hl7.org/CodeSystem/observation-category'
+                                                               )
+
+          return extract_profile('ObservationSexualOrientation') if suite_options[:us_core_version] == 'us_core_5' &&
+                                                                    observation_contains_code(resource, '76690-7')
+
+          return extract_profile('ObservationSocialHistory') if suite_options[:us_core_version] == 'us_core_5' &&
+                                                                resource_contains_category(resource, 'social-history',
+                                                                                           'http://terminology.hl7.org/CodeSystem/observation-category')
+
+          return extract_profile('ObservationSdohAssessment') if suite_options[:us_core_version] == 'us_core_5' &&
+                                                                 resource_contains_category(resource, 'sdoh',
+                                                                                            'http://terminology.hl7.org/CodeSystem/observation-category') && # rubocop:disable Layout/LineLength
+                                                                 resource_contains_category(resource, 'survey', 'http://terminology.hl7.org/CodeSystem/observation-category') # rubocop:disable Layout/LineLength
+
+          return extract_profile('ObservationSurvey') if suite_options[:us_core_version] == 'us_core_5' &&
```

Comment:
I think we should combine ObservationSdohAssessment and ObservationSurvey. see my post in slack

Reply:
All set